### PR TITLE
{lib}[GCCcore/10.3.0] scikit-build v0.11.1 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/af0f4010987446c7204009b976b00ec4 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/ef1f0c396151370f76b546a44f1719c3 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/10b713609f3bc20e377aade76f3e46e2 for a full test report.

File: easybuild/easyconfigs/s/scikit-build/scikit-build-0.11.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version = '0.11.1'
+
+homepage = 'https://scikit-build.readthedocs.io/en/latest'
+description = """Scikit-Build, or skbuild, is an improved build system generator
+for CPython C/C++/Fortran/Cython extensions."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
```
Comment (verdurin) [inline easybuild/easyconfigs/s/scikit-build/scikit-build-0.11.1-GCCcore-10.3.0.eb:13]: @Micket Weren't we talking about changing this to use `GCC` instead? Or is that not agreed yet?
I remember there was a suggestion of avoiding this annoying `binutils` dep...

Comment (Micket) [reply easybuild/easyconfigs/s/scikit-build/scikit-build-0.11.1-GCCcore-10.3.0.eb:13]: well we are already doing the good old binutils 2.36.1 in 172 other easyconfigs for GCCcore 10.3.0, so I didn't want to make 1 exception to that pattern.

Comment (verdurin) [reply easybuild/easyconfigs/s/scikit-build/scikit-build-0.11.1-GCCcore-10.3.0.eb:13]: Fair enough.

**Comment by verdurin** [comment]
Going in, thanks @Micket!

**Pull request merged**